modules/ROOT/pages/security-service.adoc

```diff
@@ -0,0 +1,23 @@
+// start {partialsdir}/attributes.adoc[]
+include::_partials/attributes.adoc[]
+// end {partialsdir}/attributes.adoc[]
+
+:toclevels: 4
+
+:toc:
+
+= Using the {mobile-security-service} Service
+
+[[introduction]]
+
+== Prerequisites
+
+* You are running OpenShift with {product-name} as described in xref:getting-started.adoc[Setting up AeroGear {product-name} on OpenShift].
+* You have provisioned Mobile Developer Console as described in link:{installation-link}[Provisioning Mobile Developer Console].
+
+[[setting-up-the-mobile-security-service]]
+== Setting Up the {mobile-security-service} Service
+
+// start {partialsdir}/generic-provisioning.adoc[]
+include::_partials/generic-provisioning.adoc[]
```

Comment:
As part of productization, we've removed some of the nested includes, (rats nest of includes). Turns out we don't have a very generic provisioning process. I'd recommend that you create a new file here, check IDM for recommended file layout

Reply:
let's handle this after